tests/functional/smoke/modes.sh

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env bash
```

Comment:
Do we need this still with the functional test?

Reply:
Not sure, copied from others.